bindata/network/ovn-kubernetes/004-config.yaml

```diff
@@ -20,3 +20,12 @@ data:
     [gateway]
     mode=local
     nodeport=true
+
+{{ if .OVNHybridOverlayEnable }}
```

Comment:
`OVNHybridOverlayEnable` is currently added to the template data as a `string`, so I don't think this will work; but you can just change it in `pkg/network/ovn_kubernetes.go` to set the value to a boolean `true` rather than a string `"true"`. (And likewise `false` instead of `""` in the unset case.)

Reply:
Actually... now that I think about it, it _will_ work this way, because `"true"` is a non-empty string, so it counts as `true`, and `""` is an empty string, so it counts as `false`, so you get the right result. But like, if the code was setting `OVNHybridOverlayEnable` to `"false"` in the false case, then this would break. So it would be clearer to use real boolean variables there.

Reply:
done